components/collector/src/source_collectors/file_source_collectors/generic.py

```diff
@@ -0,0 +1,23 @@
+"""Generic metrics collector."""
+
+from base_collectors import JSONFileSourceCollector
+from collector_utilities.functions import md5_hash
+from source_model import Entity, SourceMeasurement, SourceResponses
+
+
+class GenericSecurityWarnings(JSONFileSourceCollector):
+    """Generic collector for security warnings."""
+
+    async def _parse_source_responses(self, responses: SourceResponses) -> SourceMeasurement: # skipcq: PY-D0003
+        entities = []
+        for response in responses:
+            json = await response.json(content_type=None)
+            vulnerabilities = json.get("vulnerabilities", [])
+            for vulnerability in vulnerabilities:
+                key = md5_hash(f'{vulnerability["title"]}:{vulnerability["description"]}')
+                entities.append(
+                    Entity(
+                        key=key, title=vulnerability["title"], description=vulnerability["description"],
+                        severity=vulnerability["severity"])
+                        )
```

Comment:
Please add the bracket to the previous line to make it consistent with the rest of the code base.